### DISCUSSION TITLE
A question about the big map can not be opened in Obsidian

### DISCUSSION THREAD

**Discussion by zhubib1234** [javalent/obsidian-leaflet#80] [Q&A]
A map larger than 70M that can't be opened in Obsidian.

**Comment by valentine195** [comment]
This is likely a result of the Javascript memory heap running out.

I will not be able to program in support for this, you will have to downscale your image.

**Reply by zhubib1234** [reply]
I see. Thank you!

**Reply by zhubib1234** [reply]
Can it be solved by supporting tile map?

**Reply by valentine195** [reply]
That would require an image tiler, which (from what I can find online) has not been created in javascript yet, and I do not have the time to write one (if it is even possible).

**Reply by zhubib1234** [reply]
okay, thank you.

**Reply by Moonbase59** [reply]
Phew, a 70 MB map! :grin: One day, we’ll break Electron :rofl:
